scripts/docker-compose.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
```

Comment:
None of the documentation point to where/how to use the docker-compose files.

1. When do these get added / used?
2. If it requires additional steps, can you document i?

Reply:
They are explained itself and also on [README.md](https://github.com/pokt-network/pocketdex/pull/7/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R77)
But if you think that is not enough, please open a PR against this one or an issue to enhance the documentation.